src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -800,7 +800,9 @@ private bool ShouldIgnoreProcessorAffinityChanges(int cores, LicenseLimits licen
                                          $"Total utilized cores: {licenseLimits.TotalUtilizedCores}. " +
                                          $"Max licensed cores: {LicenseStatus.MaxCores}";
                     }
-                    Console.WriteLine(basicMessage + " " + licenseMessage);
+
+                    if (Logger.IsOperationsEnabled)
```

Comment:
Please revert this. This is used only in tests and should be written to console.